tutorials/09-variational-inference/09_variational-inference.jmd

```diff
@@ -753,16 +792,13 @@ Test set:
     OLS loss: $ols_loss2")
 ```
 
-```julia; echo=false
-# Verify the loss on the test set.
-@assert vi_loss2 < 0.01
-@assert bayes_loss2 < 0.000001
-```
-
```

Comment:
Can we keep this? It is much easier to update and rerun tutorials with these checks (not visible for readers):
````suggestion
```julia; echo=false
# Verify the loss on the test set.
@assert vi_loss2 < 0.01
@assert bayes_loss2 < 0.000001
```

````

Reply:
Good idea. So `echo=false` sets no output but will evaluate the block; `evaluate=False` will skip the evaluation?

Reply:
I don't remember all options but yes, `echo=false` ensures that nothing is printed but the code is evaluated.

Reply:
Great, thanks, I got it. Two options: `echo` and `eva` for the code blocks.

Reply:
The assertions are not recovered on Github yet.